templates/step/edit.html.ep

```diff
@@ -14,6 +14,12 @@
     setupResultButtons();
 % end
 
+% if(@$error_messages) {
+    <div class="alert-warning alert fade in" id="editor_warnings"><button class="close" data-dismiss="alert">x</button>
+        <span><%== '<ul><li>'.join('</li><li>', @$error_messages).'</li></ul>' %></span>
```

Comment:
Error messages might contain HTML syntax to allow formatting. This means of course that we must be careful when generating the messages. Currently only $needlename, $distribution and $dversion go into the messages which should be safe without further escaping? If not I can change it when I'm at the office again.